packages/opencensus-core/test/test-view.ts

```diff
@@ -180,6 +181,59 @@ describe('BaseView', () => {
        });
   });
 
+  describe('getMetric()', () => {
+    const measurementValues = [1.1, 2.3, 3.2, 4.3, 5.2];
+    const buckets = [2, 4, 6];
+    const tags: Tags = {testKey1: 'testValue', testKey2: 'testValue'};
+    const aggregationType = AggregationType.DISTRIBUTION;
+    const view: View = new BaseView(
+        'test/view/name', measure, aggregationType, ['testKey1', 'testKey2'],
+        'description test', buckets);
+    for (const value of measurementValues) {
+      const measurement = {measure, tags, value};
+      view.recordMeasurement(measurement);
+    }
+    const {descriptor, timeseries} = view.getMetric();
+
+    it('should has descriptor', () => {
+      assert.ok(descriptor);
+      assert.deepStrictEqual(descriptor, {
+        description: 'description test',
+        labelKeys: [{key: 'testKey1'}, {key: 'testKey2'}],
+        name: 'test/view/name',
+        type: 6,
+        unit: '1',
+      });
+    });
+
+    const [{startTimestamp, labelValues, points}] = timeseries;
+
+    it('should has timeseries startTimestamp', () => {
+      assert.ok(startTimestamp);
+      assert.equal(typeof startTimestamp.nanos, 'number');
```

Comment:
I think it's better to mock TimeStamp and check in tests.

Reply:
Done. Thank you